pelican/plugins/sitemap/sitemap.py

```diff
@@ -11,10 +11,7 @@
 log = logging.getLogger(__name__)
 
 XML_HEADER = """<?xml version="1.0" encoding="utf-8"?>
-<urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd"
-xmlns:xhtml="http://www.w3.org/1999/xhtml"
```

Comment:
Html namespace is still used further below:
https://github.com/pelican-plugins/sitemap/blob/4b4fbd419d717e25c3f1169a23a47b08df3dbd7d/pelican/plugins/sitemap/sitemap.py#L28

It is referenced here: https://developers.google.com/search/docs/specialty/international/localized-versions#sitemap

Reply:
Hmm, maybe not. Let's just close this pull request.